storey/utils.py

```diff
@@ -344,3 +345,25 @@ def find_filters(partitions_time_attributes, start, end, filters, filter_column)
         filters,
         filter_column,
     )
+
+
+def get_combined_filters(datetime_filters=None, filters=None):
```

Comment:
```suggestion
def combine_filters(datetime_filters, filters):
```

Reply:
I prefer to remain the None as default value. Name changed.